setup.py

```diff
@@ -46,14 +50,15 @@ def _read_contents(file_to_read):
     maintainer_email="paul.spangler@ni.com, alex.weaver@ni.com",
     keywords=["nisystemlink", "systemlink"],
     license="MIT",
-    packages=find_packages(exclude=["examples", "tests"]),
+    packages=packages,
     install_requires=[
         'aenum;python_version<"3.6"',
         "events",
         'httpx;python_version>="3.6"',
         'requests;python_version<"3.6"',
         "typing-extensions",
     ],
+    setup_requires=["setuptools>=40.1.0"],
```

Comment:
This is discouraged in favor of a `pyproject.toml` value, but this repo does not have a `pyproject.toml`. If you would prefer that I add one, please let me know.

Reply:
So now that we have a pyproject.toml, do we still need this too?

Reply:
Don't believe so. I'll double check that the dependency is still there.